WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderPostActions.java

```diff
@@ -67,6 +71,7 @@ public static boolean performLikeAction(final ReaderPost post,
         boolean isCurrentlyLiked = ReaderPostTable.isPostLikedByCurrentUser(post);
         if (isCurrentlyLiked == isAskingToLike) {
             AppLog.w(T.READER, "post like unchanged");
+            EventBus.getDefault().post(OnPostLikeUnChanged.INSTANCE);
```

Comment:
New post on eventbus

Reply:
I'd consider including blog+post id even for `OnPostLikeUnChanged` and `OnPostLikeFailure` so we can easily handle multiple calls in parallel. Wdyt?

Reply:
100% agree - and there is a todo in here somewhere with that exact thought. 👍

Reply:
in commit aec3678